scheduler/scripts/run_greedymax.py

```diff
@@ -159,16 +159,19 @@
         blueprint=optimizer_blueprint
     )
 
+    plans = optimizer.schedule()
+    print_plans(plans)
+    print('')
+
     # Timeline tests
     for tl in optimizer_blueprint.algorithm.timelines:
-        print(f'Night {tl.night}')
+        print(f'Night {tl.night + 1}')
         #     for site, ne in gm_optimizer.night_events.items():
         for site in optimizer.night_events.keys():
```

Comment:
When iterating like this over a dictionary, `.keys()` is never necessary: it always iterates over the keys unless you specify `.values()` or `.items()`.